schema/revision.shacl

```diff
@@ -0,0 +1,58 @@
+@prefix sh: <http://www.w3.org/ns/shacl#>.
```

Comment:
Really like the Shaql rules 😄 

Add one for the additional document info, i.e. report:mel?

Reply:
Lets take that in a separate ontology